### PR TITLE
Toptier v2

### PR DESCRIPTION
**Pull request by russellbodine**
Gist approved by frontend:
- [x] https://gist.github.com/russellbodine/7fd4d1a58d36eea09c79fcc2f81153a5

### CODE REVIEW

File: usaspending_api/references/urls_v2.py [hunk 1]
```diff
@@ -11,5 +11,6 @@
 
 
 urlpatterns = [
-    url(r'^agency/(?P<pk>[0-9]+)/$', agency.AgencyViewSet.as_view())
+    url(r'^agency/(?P<pk>[0-9]+)/$', agency.AgencyViewSet.as_view()),
+    url(r'^toptier_agency/$', toptier_agency.ToptierAgencyViewSet.as_view())
```
Comment (nmonga91) [inline usaspending_api/references/urls_v2.py]: If we're returning a list, is it better to pluralize the endpoint to indicate many results?

File: usaspending_api/references/views_v2/toptier_agency.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
+from django.db.models import F, Sum
+from usaspending_api.references.models import Agency, OverallTotals
+from usaspending_api.submissions.models import SubmissionAttributes
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from usaspending_api.common.views import DetailViewSet
+from usaspending_api.accounts.models import AppropriationAccountBalances
+
+
+class ToptierAgencyViewSet(APIView):
```
Comment (nmonga91) [inline usaspending_api/references/views_v2/toptier_agency.py]: Same pluralization question as above

File: usaspending_api/references/urls_v2.py (hunk 1 above)
Comment (russellbodine) [inline usaspending_api/references/urls_v2.py]: sounds good!

File: usaspending_api/references/views_v2/toptier_agency.py (hunk 2 above)
Comment (russellbodine) [inline usaspending_api/references/views_v2/toptier_agency.py]: done!

**Pull request merged**